www/docs/content/recipes/commerce-automation.mdx

```diff
@@ -0,0 +1,655 @@
+import DocCard from '@theme/DocCard';
+import Icons from '@theme/Icon';
+
+# Commerce automation
+
+This document guides you on how you can implement different forms of commerce automation within your Medusa project.
+
+## Overview
+
+Commerce automation is essential for businesses that want to save costs, provide better user experience, and avoid manual, repetitive tasks that can lead to human errors. Businesses can utilize it in different domains, including marketing, customer support, order management, and more.
+
+Medusa provides you with the necessary architecture and resources to implement commerce automation related to order management, customer service, and more. Mainly, it utilizes an event bus that allows you to listen to certain event triggers and perform asynchronously actions.
+
+This document gives a high-level overview of how you can implement different types of commerce automation.
+
+---
+
+## Re-Stock Notifications
+
+Customers may be interested in a product that is currently out of stock.
+
+Medusa already provides a [Re-stock notifications plugin](../plugins/other/restock-notifications.md). You can also implement this by creating custom entities, triggering custom events, and handling these events with a subscriber.
+
+<DocCardList colSize={6} items={[
+  {
+    type: 'link',
+    href: '/plugins/other/restock-notifications',
+    label: 'Restock Notifications Plugin',
+    customProps: {
+      icon: Icons['bolt-solid'],
+      description: 'Learn how to install the Restock Notification plugin.',
+    }
+  },
+  {
+    type: 'link',
+    href: '/development/entities/create',
+    label: 'Create an Entity',
+    customProps: {
+      icon: Icons['academic-cap-solid'],
+      description: 'Learn how to create a custom entity in the Medusa backend.',
+    }
+  },
+  {
+    type: 'link',
+    href: '/development/events/#emitting-events',
+    label: 'Emit Events',
+    customProps: {
+      icon: Icons['academic-cap-solid'],
+      description: 'Learn how to emit a custom event in the Medusa backend.',
+    }
+  },
+  {
+    type: 'link',
+    href: '/development/events/create-subscriber',
+    label: 'Create a Subscriber',
+    customProps: {
+      icon: Icons['academic-cap-solid'],
+      description: 'Learn how to create a subscriber that handles events.',
+    }
+  },
+]} />
+
+---
+
+## Automated Customer Support
+
+Customer support is essential to build a store’s brand and customer loyalty. This can include integrating with third-party services or automating notifications sent to customers when changes happen related to their orders, returns, exchanges, and more.
+
+Using Medusa’s Notification Service, you can handle notifications triggered from actions on the customer’s orders or profile.
+
+For example, when the admin adds a note to the order, this triggers the `note.created` event. You can create a Notification Service that handles this event and sends the customer an email with the note added. 
```

Comment:
This is probably not something we would want to advice since the notes are generally considered "internal"